nf_core/pipeline-template/{{cookiecutter.name_noslash}}/docs/usage.md

```diff
@@ -182,12 +173,17 @@ If you prefer, you can specify the full path to your reference genome when you r
 --fasta '[path to Fasta reference]'
 ```
 
-## Job Resources
+### `--igenomesIgnore`
+Do not load `igenomes.config` when running the pipeline. You may choose this option if you observe clashes between custom parameters and those supplied in `igenomes.config`.
+
+## Job resources
 ### Automatic resubmission
 Each step in the pipeline has a default set of requirements for number of CPUs, memory and time. For most of the steps in the pipeline, if the job exits with an error code of `143` (exceeded requested resources) it will automatically resubmit with higher requests (2 x original, then 3 x original). If it still fails after three times then the pipeline is stopped.
 
 ### Custom resource requests
-Wherever process-specific requirements are set in the pipeline, the default value can be changed by creating a custom config file. See the files in [`conf`](../conf) for examples.
+Wherever process-specific requirements are set in the pipeline, the default value can be changed by creating a custom config file. See the files hosted at [`nf-core/configs`](https://github.com/nf-core/configs/tree/master/conf) for examples.
+
+If you are likely to be running `nf-core` pipelines regularly it may be a good idea to request that your custom config file is uploaded to the `nf-core/configs` git repository. Before you do this please can you test that the config file works with your pipeline of choice using the `-c` parameter (see definition below). You can request your custom config file to be added in the `pipelines` chat on the Slack messenger for `nf-core` (may need to be invited first!) or on the `nf-core/Lobby` group on Gitter.
```

Comment:
No need to mention Gitter. Perhaps instead say that people can create a pull request with their config and should join slack to become a member of nf-core and ask for help..

Reply:
Yes, we can drop gitter as we will most likely move everyone over to slack anyways...

Reply:
Don't you have to be invited to join the Slack group? Or is that likely to change?

I was anticipating the rather tedious scenario where you ask for an invite via Gitter to join Slack...

Reply:
No not anymore: https://nf-core-invite.herokuapp.com/ 

I also put this up to the Gitter channel (as top most message in the channel description) and we should add it to the main homepage...

Reply:
Might add it to the homepage today with a logo, I guess its time

Reply:
https://github.com/nf-core/nf-co.re/pull/56

Reply:
Added details in latest commit.